picard/cluster.py

```diff
@@ -72,6 +72,9 @@
     'releasecountry': 2,
     'releasetype': 10,
     'totalalbumtracks': 5,
+    'catalognumber': 60,
+    'asin': 65,
+    'barcode': 68,
```

Comment:
I am unclear how these weights were chosen or why they are so high.

Reply:
I chose these in the same way as the ones in `metadata.py` but after trying to match ~80 albums (all files tagged with only `catalognumber`), with `'catalognumber': 60,` it was able to correctly identify almost all of them (except for ~4), while with a lower value it would identify quite a few less

But all these were guesses and as I am not familiar with the codebase yet they may very well be wrong (even tho they worked for my specific use case)